webpack.config.js

```diff
@@ -0,0 +1,58 @@
+const path = require("path")
+const webpack = require("webpack")
+const dotenv = require('dotenv')
+
+module.exports = () => {
+
+  const env = dotenv.config().parsed
+  let envKeys = []
+  if (env) {
+    envKeys = Object.keys(env).reduce((prev, next) => {
+      prev[`process.env.${next}`] = JSON.stringify(env[next])
+      return prev
+    }, {})
+  }
```

Comment:
You declare `envKeys` as an array on line 8, then assign it as an object on line 10. No bug but it doesn't help with readability.

Reply:
Yes thanks. I made sure its consistently an array.